lib/blocks.php

```diff
@@ -93,3 +93,85 @@ function do_blocks( $content ) {
 	return $content_after_blocks;
 }
 add_filter( 'the_content', 'do_blocks', 9 ); // BEFORE do_shortcode() and wpautop().
+
+/**
+ * Extract the blocks from post content for the REST API post response.
+ *
+ * @since 1.1.0
+ *
+ * @param string The post content.
+ *
+ * @return array Array of block data.
+ */
+function get_block_data_for_api_from_post_content( $content ) {
```

Comment:
Maybe `gutenberg_add_blocks_to_post_resource()`? This won't be used apart from the filter, so should probably be a prefixed name rather than a convenient one.

Reply:
fixed in https://github.com/WordPress/gutenberg/pull/2649/commits/36f1000ffa6dab2edd7ee65bf58aa8906f117a85